heartbeat/process/monitor.go

```diff
@@ -230,8 +230,6 @@ func (m *Monitor) loadHeartbeatsFromStorer(pubKey string) (*heartbeatMessageInfo
 	receivedHbmi := m.convertFromExportedStruct(*heartbeatDTO, m.maxDurationPeerUnresponsive)
 	receivedHbmi.getTimeHandler = m.timer.Now
 	crtTime := m.timer.Now()
-	crtDuration := crtTime.Sub(receivedHbmi.lastUptimeDowntime)
-	crtDuration = maxDuration(0, crtDuration)
```

Comment:
Unused variable not spotted by the compiler? :)

Reply:
spotted by the linter, actually. :)